book/content/learning/guide-to-raising-issues.md

```diff
@@ -0,0 +1,64 @@
+# How to raise an issue
+
+The standard way of asking questions, reporting bugs or requesting features is to raise an issue on the relevant repository. You can raise issues at the `Issues` tab in the upper bar of any repository 
+
+
+![alt text](../../figures/github_issues.png)
+
+
+To open a new issue, simply clickl the green `New issue` button, which will open the following screen
+
+![alt text](../../figures/github_raise_issue.png)
+
+Here you can give the issue a title and write a description. These descriptions are often standardized across a repository. We will stick to a standarized issue template of our own.
+
+To ask a question you can open a new issue and copy/fill in the issue template below. Copy the raw markdown text rather than copy pasting the rendered text. Before you submit a new issue, you may preview the description using the `Preview` button to check whether pictures, links etc. have rendered properly. Also, make sure to assign the relevant people on your new issue, else it may lie dorment in perpetuity! 
```

Comment:
```suggestion
To ask a question you can open a new issue and copy/fill in the issue template below. Copy the raw markdown text rather than copy pasting the rendered text. Before you submit a new issue, you may preview the description using the `Preview` button to check whether pictures, links etc. have rendered properly. Also, make sure to assign the relevant people (for our course, tag at least someone of the teaching team) on your new issue, else it may lie dormant in perpetuity! 
```